crates/biome_js_analyze/tests/specs/nursery/useIterableCallbackReturn/valid.js

```diff
@@ -0,0 +1,162 @@
+[].forEach((a) => {
```

Comment:
```suggestion
/* should not generate diagnostics */
[].forEach((a) => {
```

Make sure to add this comment to files that don't need to raise diagnostics, so our infra will panic in case it does

Reply:
Done